source/includes/stackpath/_sites.md

```diff
@@ -365,4 +365,119 @@ Disable Serverless Scripts for a given site.
 Attributes | &nbsp;
 ------- | -----------
 `taskId` <br/>*string* | The task id related to the site deletion.
-`taskStatus` <br/>*string* | The status of the operation.
\ No newline at end of file
+`taskStatus` <br/>*string* | The status of the operation.
+
+<!-------------------- List WAF Setting for a Site -------------------->
+
+### Retrieve WAF Settings for a Site
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/wafsettings/1b1cd7e6-41ab-4e0f-a59a-5c4b89da1b36"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+   "data":{
+      "siteId":"1b1cd7e6-41ab-4e0f-a59a-5c4b89da1b36",
+      "stackId":"1415650d-1d02-4097-a79a-8f6e4bb4f483",
+      "apiUrls":[
+         "/test/this/way"
+      ],
+      "ddosSetting":{
+         "globalThreshold":"5000",
```

Comment:
Should have spaces between labels and values

Reply:
my json formatter messed it up. Thanks, I'll update it